Library/Formula/datetime-fortran.rb

```diff
@@ -0,0 +1,32 @@
+class DatetimeFortran < Formula
+  desc "Fortran time and date manipulation library"
+  homepage "https://github.com/milancurcic/datetime-fortran"
+  url "https://github.com/milancurcic/datetime-fortran/releases/download/v1.4.2/datetime-fortran-1.4.2.tar.gz"
+  sha256 "5b70c6e5d38032951e879b437e9ac7c5d483860ce8a9f6bbe6f1d6cd777e737f"
+
+  head do
+    url "https://github.com/milancurcic/datetime-fortran.git"
+
+    depends_on "autoconf"   => :build
+    depends_on "automake"   => :build
+    depends_on "pkg-config" => :build
+  end
+
+  option "without-test", "Run build time tests"
```

Comment:
I always get confused about phrasing the `"without-..."` descriptions. Should I `s/Run build time tests/Skip build time tests/`?